batavia/modules/time.js

```diff
@@ -191,21 +193,28 @@ time.gmtime = function(seconds) {
```

Comment:
At column 16: [(no-trailing-spaces) Trailing spaces not allowed.](http://eslint.org/docs/rules/no-trailing-spaces)